book/templating.rst

```diff
@@ -51,6 +51,12 @@ template - a text file parsed by PHP that contains a mix of text and PHP code:
         </body>
     </html>
 
+.. caution::
+
+    The use of PHP templates is strongly discouraged for Symfony applications
+    because its support will be dropped in future Symfony versions. Create all
+    your templates using the Twig templating language.
```

Comment:
To not confuse the reader I suggest to move this after the introduction of Twig.